integration/spark/app/integrations/container/deltaMergeCommandVerification.json

```diff
@@ -0,0 +1,124 @@
+{
+  "producer": "https://github.com/OpenLineage/OpenLineage/tree/1.21.0-SNAPSHOT/integration/spark",
+  "schemaURL": "https://openlineage.io/spec/2-0-2/OpenLineage.json#/$defs/RunEvent",
+  "eventType": "COMPLETE",
+  "inputs": [],
+  "outputs": [
+    {
+      "namespace": "file",
+      "name": "/tmp/delta/events",
+      "facets": {
+        "dataSource": {
+          "_producer": "https://github.com/OpenLineage/OpenLineage/tree/1.21.0-SNAPSHOT/integration/spark",
```

Comment:
I would recommend removing `producer` and `schemaURL` fields' values. It's not related to the issue and may result in problems in future versions.